lib/src/main/java/io/ably/lib/util/AblyError.java

```diff
@@ -0,0 +1,160 @@
+package io.ably.lib.util;
+
+public class AblyError {
```

Comment:
Can we use className as `AblyErrorCode` instead of `AblyError` ? 
Because, this class only contains integer error codes.